pkg/cel/policies/vpol/engine/provider.go

```diff
@@ -80,7 +90,9 @@ func NewKubeProvider(
 	polexEnabled bool,
 ) (Provider, error) {
 	reconciler := newReconciler(compiler, mgr.GetClient(), polexLister, polexEnabled)
-	builder := ctrl.NewControllerManagedBy(mgr).For(&policiesv1alpha1.ValidatingPolicy{})
+	builder := ctrl.NewControllerManagedBy(mgr).
+		For(&policiesv1alpha1.ValidatingPolicy{}).
+		Watches(&policiesv1alpha1.NamespacedValidatingPolicy{}, &handler.EnqueueRequestForObject{})
```

Comment:
Not sure this is the correct way to watch multiple resources 🤔 
Shall we create one controller per resource type and use the same reconciler ?